test/pkg/installers/rhpam_kogito_installer.go

```diff
@@ -89,13 +90,28 @@ func GetRhpamKogitoInstaller() (installers.ServiceInstaller, error) {
 func installRhpamKogitoUsingYaml() error {
 	framework.GetMainLogger().Info("Installing RHPAM Kogito operator")
 
+	// Create namespace first so ImageStream can be placed there
+	if err := framework.CreateNamespace(rhpamKogitoNamespace); err != nil {
+		return err
+	}
+
 	yamlContent, err := framework.ReadFromURI(config.GetOperatorYamlURI())
 	if err != nil {
 		framework.GetMainLogger().Error(err, "Error while reading kogito-operator.yaml file")
 		return err
 	}
 
-	yamlContent = strings.ReplaceAll(yamlContent, "quay.io/kiegroup/rhpam-kogito-operator:"+version.Version, framework.GetOperatorImageNameAndTag())
+	// Use insecure ImageStream when deploying on OpenShift to support using insecure registries
+	if framework.IsOpenshift() {
+		if err := framework.CreateInsecureImageStream(rhpamKogitoNamespace, rhpamKogitoImageStreamName, config.GetOperatorImageTag(), framework.GetOperatorImageNameAndTag()); err != nil {
+			return err
+		}
+
+		rhpamKogitoInternalImageTagName := fmt.Sprintf("image-registry.openshift-image-registry.svc:5000/%s/%s:%s", rhpamKogitoNamespace, rhpamKogitoImageStreamName, config.GetOperatorImageTag())
```

Comment:
shouldn't it be configurable?

Reply:
Do you mean `image-registry.openshift-image-registry.svc:5000`? That is OCP internal registry URL, it should be same everywhere.